src/frontend/src/icp/utils/icp-account.utils.ts

```diff
@@ -54,3 +55,25 @@ export const getIcrcv2AccountIdString = (accountId: Icrcv2AccountId): string =>
 
 	return assertNever({ variable: accountId, typeName: 'Icrcv2AccountId' });
 };
+
+export const parseIcrcAccountToAccountIdentifierText = (address: Address): string | undefined => {
```

Comment:
- maybe `tryToParse...` since we ignore errors?

- can you add JSDocs for providing context? Since I asked few question on Slack, might be useful maybe?